pyproject.toml

```diff
@@ -29,6 +29,14 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
   "bioio-base>=0.1.1",
+  "dask[array]>=2021.4.1",
```

Comment:
Lots of dependencies added here that are specific to the `writers`. Does it seem worth it to move these to an extra so they are installable like `bioio[writers]` instead of by default or is that just added complexity for not much gain?

Reply:
dask is an interesting one - I can see many projects importing dask on their own too.  Personally I don't have a strong opinion other than "keep it simple" - start with this and we can change things later if we need to separate it into an extra.